loopTestApp/Db/Makefile

```diff
@@ -17,10 +17,11 @@ include $(TOP)/configure/RULES
 #----------------------------------------
 #  ADD RULES AFTER THIS LINE
 
-example.template: example.generated
+../O.Common/example.db: ../example.generated
```

Comment:
Well, I wouldn't put generated code in a source directory.
That script seems to be ARCH-independent, so should go into `O.Common`, shouldn't it?